modules/otel-troubleshoot-metrics.adoc

```diff
@@ -19,26 +19,47 @@ The Operator creates a `<cr_name>-collector-monitoring` telemetry service that y
 
 .Procedure
 
-. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource:
+. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource (CR):
 
 +
 [source,yaml]
 ----
+# ...
   config: |
     service:
       telemetry:
         metrics:
           address: ":8888" # <1>
+# ...
 ----
 <1> The address at which the internal collector metrics are exposed. Defaults to `:8888`.
 
-// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
-
 . Retrieve the metrics by running the following command, which uses the port-forwarding Collector pod:
 +
 [source,terminal]
 ----
 $ oc port-forward <collector_pod>
 ----
 
-. Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors` depending on the deployment mode of the OpenTelemetry Collector:
+:
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+spec:
+# ...
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true # <1>
+# ...
+----
++
+[NOTE]
+====
+Alternatively, if you don't set the `enableMetrics` field to `true`, you can access the metrics endpoint at `+http://localhost:8888/metrics+`.
+====
+
+. In the *Observe* section of the OpenShift Console, enable *User Workload Monitoring* to visualize the scraped metrics. You can filter by metrics from an OpenTelemetry Collector instance or OpenTelemetry components such as processors, receivers, or exporters.
```

Comment:
We need to add a note here that Data points for Processor Metrics accepted, dropped, refused / second and Processor Spans accepted, dropped, refused / second depends on the type of processor used in the OpenTelemetry collector config. Not all processors expose the required metrics. Refer bug https://issues.redhat.com/browse/TRACING-4484

Reply:
Done. Tank you.